app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] A very vital requirement for this milestone is that the application should ask for confirmation of email. So, please, let's enable `:confirmable` in the user model here ☝️